packages/cli/test/integration/credentials.test.ts

```diff
@@ -124,12 +124,10 @@ describe('POST /credentials', () => {
 	});
 
 	test('should fail with invalid inputs', async () => {
-		await Promise.all(
```

Comment:
using `await Promise.all(` is somehow causing DB connection resets, leading to failing tests.
Not sure why that is, but switched to a for loop to unblock the CI.